float64 pan
float64 tilt
float64 rotation
float64 x
float64 y